tests/back/api/test_dataset_api.py

```diff
@@ -92,17 +88,25 @@ def test_get_wrong_dataset(client: TestClient):
     assert response.text == '{"detail":"Dataset not found"}'
 
 
-def test_modify_dataset(client: TestClient):
+def test_get_types(client: TestClient):
+    response = client.get("/api/v1/dataset/types/2")
+    data = response.json()
+    assert data["SepalLengthCm"]["type"] == "Value"
+    assert data["SepalLengthCm"]["dtype"] == "float64"
+    assert data["Species"]["type"] == "Value"
+    assert data["Species"]["dtype"] == "string"
```

Comment:
Here I would better use the whole answer as a dictionary.
pytest has a good diff engine to show that it is different in the actual and expected values :).

```suggestion
    assert data == {
        "SepalLenghtCm" {"type": "Value", "dtype": "float64"}, 
        "SepalWidth": ...}
```